setup.py

```diff
@@ -35,11 +36,16 @@
 #   - https://github.com/python/mypy
 class build_py(_build_py):
     def pin_version(self):
-        path = os.path.join(self.build_lib, "dvc")
-        self.mkpath(path)
-        with open(os.path.join(path, "version.py"), "w") as fobj:
-            fobj.write("# AUTOGENERATED at build time by setup.py\n")
-            fobj.write(f'__version__ = "{version}"\n')
+        ver_file = Path(self.build_lib) / "dvc" / "version.py"
```

Comment:
Maybe just the `file` is enough?

Reply:
idk lol; not even the point of this PR; darn it `pylint`

Reply:
Pyint started complaining about it in this PR? If so, just add flags to ignore it in these lines. Let's not change unrelated logic, please.

Reply:
I assume the `pytlint` CI is currently configured to complain about touched files in their entriety so if you don't want PRs to slowly fix unrelated issues then isn't it better to switch off that option?

I'd prefer to either

- revert in entirety & merge with failing pylint, or
- merge as-is including tangential fixes

in contrast AFAIK

- reverting fixes, adding tangential ignore directives, and merging

is the worst of all combinations unless I'm missing something?

Reply:
@casperdcl was pre-commit ci failing or our `Tests / lint`?

Reply:
`pre-commit ci` as well as local `pre-commit` hook.